src/ui-pager/angular/pager-items-comp.ts

```diff
@@ -86,7 +85,7 @@ export abstract class TemplatedItemsComponent implements DoCheck, OnDestroy, Aft
         if (value instanceof ObservableArray) {
             needDiffer = false;
         }
-        if (needDiffer && !this._differ && isListLikeIterable(value)) {
+        if (needDiffer && !this._differ && Array.isArray(value)) {
```

Comment:
isListLikeIterable does a little more than just isArray(), do we need it here as well?

```
export function isListLikeIterable(obj: any): boolean {
  if (!isJsObject(obj)) return false;
  return Array.isArray(obj) ||
      (!(obj instanceof Map) &&  // JS Map are iterables but return entries as [k, v]
       Symbol.iterator in obj);  // JS Iterable have a Symbol.iterator prop
}
```